### PR TITLE
Add Godot Engine SDK to Community SDKs

### PR DESCRIPTION
**Pull request by fenix-hub**
## What does this PR do?
This PR will mention [Godot Engine](https://godotengine.org/) community-maintained SDK in the README

## Test Plan
No test plan required.

## Related PRs and Issues
[Implement Godot SDK](https://github.com/appwrite/sdk-generator/issues/276#issuecomment-934114150)
[Mention SDK](https://github.com/appwrite/awesome-appwrite/pull/365#issuecomment-955645247)

### Have you read the [Contributing Guidelines on issues](https://github.com/appwrite/appwrite/blob/master/CONTRIBUTING.md)?
Yes, I did

### CODE REVIEW

**Comment by eldadfux** [comment]
Great work!

File: README.md [hunk 1]
```diff
@@ -139,8 +139,9 @@ Below is a list of currently supported platforms and languages. If you wish to h
 * ✅  &nbsp; [.NET](https://github.com/appwrite/sdk-for-dotnet) - **Experimental** (Maintained by the Appwrite Team)
 
 #### Community
-* ✅  &nbsp; [Appcelerator Titanium](https://github.com/m1ga/ti.appwrite) (Maintained by [Michael Gangolf](https://github.com/m1ga/))    
-    
+* ✅  &nbsp; [Appcelerator Titanium](https://github.com/m1ga/ti.appwrite) (Maintained by [Michael Gangolf](https://github.com/m1ga/))  
+* ✅  &nbsp; [Godot Engine](https://github.com/m1ga/ti.appwrite) (Maintained by [fenix-hub @GodotNuts](https://github.com/GodotNuts/appwrite-sdk/))
```
Comment (m1ga) [inline README.md:143]: @fenix-hub I like that you link both SDKs to my repo but to make it right you should set the first link to your Godot SDK and the second link to your profile :smile:

Comment (fenix-hub) [reply README.md:143]: Oh sorry! And thanks for noticing! I just copy-pasted your line to make mine consistent, but forgot to replace my link.

Comment (m1ga) [reply README.md:143]: no problem :wink: ! saw the new PR already. Great work on the Godot SDK btw!

**Pull request merged**